README.rst

```diff
@@ -0,0 +1,51 @@
+==========
```

Comment:
Is there a benefit to using RST vs. markdown? Just wondering why the change in readme format.

Reply:
I've always used RST, because then you can directly include the README in your Sphinx docs without writing weird conversion scripts, or having redundant text. We can use md if you like.

Reply:
Nope, I'm fine with RST, was just wondering about the motivation for changing the readme format.